### DISCUSSION TITLE
2 Octopus Accounts

### DISCUSSION THREAD

**Discussion by prckmc** [BottlecapDave/HomeAssistant-OctopusEnergy#741] [General]
Hi 
Is it possible to setup 2 Octopus accounts, i have 2 properties on 2 different accounts but under the same API key but so far haven't found a way to add the 2nd account.

**Comment by markcocker** [comment]
> Is it possible to setup 2 Octopus accounts

Yes, see issue #223

**Reply by prckmc** [reply]
Thanks Mark

**Comment by BottlecapDave** [comment]
Yes, as @markcocker mentioned, the feature is currently in beta as I can't test it myself. Feedback welcome, but if everything goes well it will be included in the next feature release of the integration.

**Reply by prckmc** [reply]
Hi Dave, had a few issues downgrading to Beta version but since then it has worked well, i am monitoring Electricity and Gas in one property and Electricity and Export meter in another on 2 different accounts. All working very well.
